tox.ini

```diff
@@ -21,32 +21,25 @@ extras =
 
 [py-cover]
 commands =
-    coverage run --source=waitress --parallel-mode {envbindir}/nosetests
-    coverage combine
-    coverage xml -o {envname}.xml
+    coverage run {envbindir}/nosetests waitress
 
 extras =
     testing
 
 [testenv:py27-cover]
 commands =
     {[py-cover]commands}
-setenv =
-    COVERAGE_FILE=.coverage.py2
 
 [testenv:py35-cover]
 commands =
     {[py-cover]commands}
-setenv =
-    COVERAGE_FILE=.coverage.py3
```

Comment:
Why can these lines be removed? We still want it to generate two separate coverage files, no?

Reply:
I don't care about the separate coverage files, but I put them back.